index.html

```diff
@@ -25,6 +26,9 @@
     <script src="js/zap-common.js"></script>
     <script src="js/everything.js"></script>
     <script src="https://www.youtube.com/iframe_api"></script>
+	<script src="https://connect.soundcloud.com/sdk/sdk-3.0.0.js"></script>
+	<script src="https://cdn.plyr.io/1.5.16/plyr.js"></script>
```

Comment:
We should be using bower. `bower install --save plyr`, then the `src` should look more like the other bower components above. While we're at it, this line should be moved up a bit so it's grouped with the others